npm/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "@unsplash/intlc",
-  "version": "0.2.0-0",
+  "version": "0.2.1-0",
```

Comment:
Will publish once the release has been created which will be once this is merged